### ISSUE TITLE
Cannot move files to trash after upgrading to Mac OSX Sonoma

### ISSUE THREAD

**Issue by landisdesign** [microsoft/vscode#207007]
Type: <b>Bug</b>

Since I've upgraded from Ventura to Sonoma, files must be deleted permanently. They can no longer be moved to trash. I'm seeing the following in the dev console:

<img width="498" alt="Screenshot 2024-03-06 at 12 26 09 PM" src="https://github.com/microsoft/vscode/assets/44303132/974b22e5-5578-4a87-9f27-686a7f2ca562">

VS Code version: Code 1.87.0 (019f4d1419fbc8219a181fab7892ebccf7ee29a2, 2024-02-27T23:42:51.279Z)
OS version: Darwin arm64 23.3.0
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Apple M1 (8 x 24)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|2, 2, 2|
|Memory (System)|8.00GB (0.10GB free)|
|Process Argv|--crash-reporter-id 1c7a3660-3d3a-4432-9355-b608d7d5e6f1|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (13)</summary>

Extension|Author (truncated)|Version
---|---|---
theme-verdandi|be5|2.0.1
markdown-preview-github-styles|bie|2.0.3
vscode-eslint|dba|2.4.4
gitlens|eam|14.8.2
prettier-vscode|esb|10.1.0
copilot|Git|1.172.0
copilot-chat|Git|0.13.0
vscode-graphql-syntax|Gra|1.3.4
vscode-scss|mrm|0.10.0
debugpy|ms-|2024.2.0
python|ms-|2024.2.1
scala|sca|0.5.7
errorlens|use|3.16.0

(2 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vspor879:30202332
vspor708:30202333
vspor363:30204092
vswsl492cf:30256860
vstes627:30244334
vscoreces:30445986
vscod805cf:30301675
binariesv615:30325510
vsaa593cf:30376535
py29gd2263:30899288
c4g48928:30535728
azure-dev_surveyone:30548225
962ge761:30959799
pythongtdpath:30769146
welcomedialog:30910333
pythonidxpt:30866567
pythonnoceb:30805159
asynctok:30898717
pythontestfixt:30902429
pythonregdiag2:30936856
pyreplss1:30897532
pythonmypyd1:30879173
pythoncet0:30885854
h48ei257:30968133
pythontbext0:30879054
accentitlementsc:30887149
dsvsc016:30899300
dsvsc017:30899301
dsvsc018:30899302
d34g3935:30971562
edj9j583:30969295
fegfb526:30973843
78248357:30976999
ccp1r3:30958155
dsvsc020:30976470

```

</details>

**Comment by bpasero** [comment]
@landisdesign unfortunately we failed to log the actual error, but I have pushed a change for that. Can you test this scenario using latest insiders and report what you get as error?

You can give our preview releases a try from: https://code.visualstudio.com/insiders/

**Comment by landisdesign** [comment]
Gahhh and, of course, now it works fine. I'm unaware of what changed since I logged this bug? I haven't had any system updates or VSCode updates that I'm aware of. Chrome has been updated, and I don't know if Electron shares code with it that might have impacted anything but I doubt it, given this was a file system issue...

Thank you so much for trying to run this down, though. I appreciate you adding logging on a quick turnaround into the insiders edition.

I'm not sure if there's anything else you might want to explore with this, but I'm happy to call this closed and write it off to the gods.
